pages/_app.js

```diff
@@ -59,6 +60,29 @@ function MyApp({ Component, pageProps }) {
 
   return (
     <>
+
+      <Head>
+        <link rel="icon" type="image/png" href="/images/favicon.ico" />
+        <link
+          rel="apple-touch-icon"
+          sizes="180x180"
+          href="/images/apple-touch-icon.png"
+        />
+        <link
+          rel="icon"
+          type="image/png"
+          sizes="32x32"
+          href="/images/favicon-32x32.png"
+        />
+        <link
+          rel="icon"
+          type="image/png"
+          sizes="16x16"
+          href="/images/favicon-16x16.png"
+        />
+        <link rel="manifest" href="/images/site.webmanifest" />
+      </Head>
+      
```

Comment:
These are better off in the Meta.js file so as to not have this information in two places.

Reply:
fixed